java/arcs/core/analysis/RecipeGraph.kt

```diff
@@ -95,17 +93,7 @@ data class RecipeGraph(
         }
 
         /** A node representing a particle. */
-        data class Particle(
-            val particle: Recipe.Particle,
-            val claims: List<Claim>,
-            val checks: List<Check>
-        ) : Node() {
-            constructor(particle: Recipe.Particle) : this(
-                particle,
-                particle.spec.claims,
-                particle.spec.checks
-            )
-
+        data class Particle(val particle: Recipe.Particle) : Node() {
```

Comment:
May be also revert the changes to the constructor of `RecipeGraph` or at least make it private? We should not allow constructing a `RecipeGraph` directly from `particleNodes` and `handleNodes`.

Reply:
Completely forgot about that, thanks